changelog/fix_message_for_lint_unused_method_argument.md

```diff
@@ -0,0 +1 @@
+* [#11322](https://github.com/rubocop/rubocop/issues/11322): Fix message for `Lint/UnusedMethodArgument`. ([@JunichiIto][])
```

Comment:
Changelog is not required. Can you remove the changelog entry?

Reply:
Okay, I removed it.